clients/java/src/main/java/io/camunda/client/impl/fetch/DocumentContentGetRequestImpl.java

```diff
@@ -63,6 +66,9 @@ public CamundaFuture<InputStream> send() {
     if (storeId != null) {
       queryParams.put("storeId", storeId);
     }
+    if (contentHash != null) {
+      queryParams.put("contentHash", contentHash);
+    }
```

Comment:
Should we make this parameter required if the verification is enabled by default in all document stores?